src/test/java/com/squareup/javapoet/TypeSpecTest.java

```diff
@@ -2424,4 +2425,19 @@ private String toString(TypeSpec typeSpec) {
     assertThat(TypeSpec.enumBuilder(className).addEnumConstant("A").build().name).isEqualTo("Example");
     assertThat(TypeSpec.annotationBuilder(className).build().name).isEqualTo("Example");
   }
+
+  @Test public void javadocWithTrailingLineDoesNotAddAnother() {
```

Comment:
What about a dedicated test for adding the newline?

Reply:
edaccf4